aws/aws-how-to/instances/find-ubuntu-images.rst

```diff
@@ -63,7 +63,7 @@ Images for EC2 and EKS
 
          ubuntu/eks/$RELEASE/$K8S_VERSION/stable/current/$ARCH/hvm/ebs-gp2/ami-id
 
-      * RELEASE: `jammy`, `22.04` (for EKS 1.29 or greater); `focal`, `20.04` (for EKS <1.29)
```

Comment:
i think this change wasn't clear enough. changing this to to (for EKS <=1.29) would be perfect. right now, this implies that focal is going to be supported in future releases, which isn't the intent.

Reply:
added another PR https://github.com/canonical/ubuntu-cloud-docs/pull/181